ChangeLog.d/rm-truncated-hmac-ext.txt

```diff
@@ -0,0 +1,3 @@
+Removals
+   * Remove MBEDTLS_SSL_TRUNCATED_HMAC config option since it is no longer
+     supported in 3.0. Addresses #4341.
```

Comment:
This sentence doesn't make sense: the option is being removed in 3.0 since it is being removed in 3.0...

Reply:
I read it differently as: the config option is being removed since truncated HMAC support is being dropped, which to me felt like somewhat different things. I have changed the wording though so please take another look.